cmake/SociConfig.cmake

```diff
@@ -69,7 +69,7 @@ else()
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
         elseif (CMAKE_SYSTEM_NAME MATCHES "Windows")
             # We need to disable the warnings for using %I64d with MinGW.
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format")
+            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format -Wno-format")
```

Comment:
@vadz Here the full command line in logs https://ci.appveyor.com/project/snikulov/soci/build/job/ltddtlpedtw4rth0 (added -DCMAKE_VERBOSE_MAKEFILE=ON)

Another question.
Is it possible to use same approach for odbc back-end as for mysql https://github.com/SOCI/soci/blob/master/src/backends/mysql/common.h#L57 
instead of sscanf ?